pkg/chains/constants.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2024 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package chains
+
+const (
+	SignedMessagesCount     = "sgcount"
+	SignsStoredCount        = "stcount"
+	PayloadUploadeCount     = "plcount"
+	MarkedAsSignedCount     = "mrcount"
+	PipelineRunSignedName   = "pipelinerun_sign_created_total"
```

Comment:
I think the `name` and the `description`  constants for metrics should be in pipelinerun/metrics or taskrun/metrics.

Reply:
I don't have a strong opinion on this. It might make sense to have them all listed in one place.